spikeinterface/extractors/neoextractors/openephys.py

```diff
@@ -63,22 +65,30 @@ def __init__(self, folder_path, stream_id=None):
         neo_kwargs = {'dirname': folder_path}
         NeoBaseRecordingExtractor.__init__(self, stream_id=stream_id, **neo_kwargs)
 
+        if hasattr(pi, "read_openephys"):
```

Comment:
Lets avoid this.
We will increase the version depency as soon as we release PI.
No ?